### ISSUE TITLE
/gang

### ISSUE THREAD

**Issue by yegor256** [zerocracy/farm#366]
Let's create a new web page `/gang`, which will take all people from `people.xml` and render in a table. For each person let's show:

  * GitHub name/link
  * `Awards.total()`
  * `Agenda.total()`
  * `Projects.total()`

Let's make the list sorted by default by the awards column.

We will definitely have performance issues here. Let's think right away how to cache the calculations.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) this job was assigned to you 8 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by g4s8** [comment]
@yegor256 I think it may produce inconsistency between reputation in `people.xml` and `awards.xml` especially after #439 PR, because awards will be calculated from last 90 days on the fly, but reputation in `people.xml` will be updated only when awards updated.

**Comment by yegor256** [comment]
@g4s8 you are right, and I think that we should not just ignore awards during selection. Instead, we should _remove_ older rewards from `awards.xml` regularly, and update `people.xml`. See? I would reject that PR.

**Comment by g4s8** [comment]
@yegor256 also we need to keep total project count for each person, I think we can add project as a link:
```xml
<links>
  <link rel="pid" href="C3NDPUA8L" />
</link>
```
and update them in `Roles.assign()`, `Roles.resign()`.
What do you think?

**Comment by g4s8** [comment]
@0crat waiting

**Comment by 0crat** [comment]
> @0crat waiting ([here](https://github.com/zerocracy/farm/issues/366#issuecomment-360707691))

@g4s8 The impediment for [#366](https://github.com/zerocracy/farm/issues/366) was registered successfully by @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by yegor256** [comment]
@g4s8 why not just a number of projects? We already have their list in `projects/yegor256.xml`. No need to reproduce the full list in `people.xml`. Just a number would be enough, I believe.

**Comment by g4s8** [comment]
@yegor256 zerocracy/datum#266 and #515 were merged, #552 todo was posted, can we close?

**Comment by 0crat** [comment]
To cancel the order use `refuse` as in §6

**Comment by yegor256** [comment]
@g4s8 thanks!

**Comment by 0crat** [comment]
Order was successfully finished: +30 points just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8), total is [+3970](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
The job [#366](https://github.com/zerocracy/farm/issues/366) is now out of scope

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#840](https://github.com/zerocracy/farm/issues/840), [#841](https://github.com/zerocracy/farm/issues/841) are still not solved; solved: [#552](https://github.com/zerocracy/farm/issues/552).

**Comment by 0pdd** [comment]
@yegor256 the puzzle [#841](https://github.com/zerocracy/farm/issues/841) is still not solved; solved: [#1059](https://github.com/zerocracy/farm/issues/1059), [#552](https://github.com/zerocracy/farm/issues/552), [#840](https://github.com/zerocracy/farm/issues/840).

**Comment by 0pdd** [comment]
@yegor256 all 4 puzzles are solved here: [#1059](https://github.com/zerocracy/farm/issues/1059), [#552](https://github.com/zerocracy/farm/issues/552), [#840](https://github.com/zerocracy/farm/issues/840), [#841](https://github.com/zerocracy/farm/issues/841).
